samples/cellular/nrf_cloud_multi_service/overlay_nrf7002ek_wifi_no_lte.conf

```diff
@@ -89,42 +128,44 @@ CONFIG_MBEDTLS=y
 CONFIG_MBEDTLS_ENABLE_HEAP=y
 CONFIG_MBEDTLS_RSA_C=y
 CONFIG_MBEDTLS_SSL_SERVER_NAME_INDICATION=y
-CONFIG_MBEDTLS_HEAP_SIZE=120000
+CONFIG_MBEDTLS_HEAP_SIZE=100000
+
+## Disable unneeded MBEDTLS features to save flash and RAM
+CONFIG_MBEDTLS_CHACHA20_C=n
+CONFIG_MBEDTLS_CHACHA20_C=n
+CONFIG_MBEDTLS_CHACHAPOLY_C=n
+CONFIG_MBEDTLS_POLY1305_C=n
+CONFIG_MBEDTLS_SHA1_C=n
+CONFIG_MBEDTLS_MAC_SHA256_ENABLED=n
+CONFIG_MBEDTLS_CIPHER_MODE_CBC=n
+CONFIG_MBEDTLS_CIPHER_MODE_CFB=n
+CONFIG_MBEDTLS_CIPHER_MODE_OFB=n
+CONFIG_MBEDTLS_SSL_SRV_C=n
+CONFIG_MBEDTLS_SSL_COOKIE_C=n
```

Comment:
Just a question: How did you find out that these are not needed?

Reply:
Mostly trial and error (after reading each one's description / usage in code to confirm it is unlikely to be used by this sample). It's not impossible that I disabled some features that would be used if enabled, but removing these has not prevented connectivity, and AFAIK the main ciphers we use are enabled.